cpp/unreal_plugins/SpEngine/Source/SpEngine/GameWorldService.h

```diff
@@ -41,6 +64,120 @@ class GameWorldService {
+            return Std::reinterpretAsVectorOf<uint64_t>(Unreal::findActors(world_));
+        });
+
+        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "find_actors_as_map", [this]() -> TMapStringToInt {
+            return getIntMapFrom(Unreal::findActorsAsMap(world_));
+        });
+
+        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "get_components", [this](const uint64_t& ptr) -> std::vector<uint64_t> {
+            return Std::reinterpretAsVectorOf<uint64_t>(Unreal::getComponents(reinterpretAs<AActor*>(ptr)));
+        });
+
+        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "get_components_as_map", [this](const uint64_t& ptr) -> TMapStringToInt {
+            return getIntMapFrom(Unreal::getComponentsAsMap(reinterpretAs<AActor*>(ptr)));
+        });
+
+        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "get_children_components", [this](const uint64_t& ptr, const bool& include_all_descendants) -> std::vector<uint64_t> {
+            return Std::reinterpretAsVectorOf<uint64_t>(Unreal::getChildrenComponents(reinterpretAs<USceneComponent*>(ptr), include_all_descendants));
+        });
+
+        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "get_children_components_as_map", [this](const uint64_t& ptr, const bool& include_all_descendants) -> TMapStringToInt {
+            return getIntMapFrom(Unreal::getChildrenComponentsAsMap(reinterpretAs<USceneComponent*>(ptr), include_all_descendants));
+        });
+
+        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "get_object_properties_as_string_from_object", [this](const uint64_t& ptr) -> std::string {
+            return Unreal::getObjectPropertiesAsString(reinterpretAs<UObject*>(ptr));
+        });
+
+        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "get_object_properties_as_string_from_struct", [this](const uint64_t& value_ptr, const uint64_t& struct_ptr) -> std::string {
+            return Unreal::getObjectPropertiesAsString(reinterpretAs<void*>(value_ptr), reinterpretAs<UStruct*>(struct_ptr));
+        });
+
+        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "set_object_properties_from_string_for_object", [this](const uint64_t& ptr, const std::string& string) -> void {
+            Unreal::setObjectPropertiesFromString(reinterpretAs<UObject*>(ptr), string);
+        });
+
+        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "set_object_properties_from_string_for_struct", [this](const uint64_t& value_ptr, const uint64_t& struct_ptr, const std::string& string) -> void {
+            Unreal::setObjectPropertiesFromString(reinterpretAs<void*>(value_ptr), reinterpretAs<UStruct*>(struct_ptr), string);
+        });
+
+        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "find_property_by_name_from_object", [this](const uint64_t& ptr, const std::string& name) -> GameWorldServicePropertyDesc {
```

Comment:
Good. This code is kind of ugly but it's the correct way to do this. Let's call the variable `game_world_service_property_desc` so it matches the class name.